db/fixtures/miq_product_features.yml

```diff
@@ -5176,6 +5176,14 @@
           :description: Add a Repository
           :feature_type: admin
           :identifier: embedded_configuration_script_source_add
+        - :name: Edit Repository
+          :description: Edit a Repository
+          :feature_type: admin
+          :identifier: embedded_configuration_script_source_edit
+        - :name: Remove Repository
+          :description: Remove a Repository
+          :feature_type: admin
+          :identifier: embedded_configuration_script_source_remove
```

Comment:
Just noticed that for `embedded_automation_manager_credentials`, they use `delete` instead of `remove`. not sure if we want / need to keep it consistent with that. (see [here](https://github.com/ManageIQ/manageiq/pull/13795/files#diff-bccee92afbf0d3d195b7bcf343b1b6c4R5198))